### PR TITLE
Document testing environment variables

### PR DESCRIPTION
**Pull request by pnax**
## Purpose

Document testing environment variables

## Context

#737

## Changes

Documentation

## How to test this PR

Documentation only

### CODE REVIEW

File: docs/EnvironmentVariables.md [hunk 1]
```diff
@@ -21,3 +21,14 @@
 * `ZM_BACKEND_RPCAPI_LOGJSON`: Setting it to any thruthy value (non-empty
   string or non-zero number) will configure the logger to log in JSON format,
   undefined by default.
+
+## Variables specific to testing
```
Comment (matsduf) [inline docs/EnvironmentVariables.md:25]: Everything is about testing. I suggest:
```
## Variables used by unit test scripts
```

File: docs/EnvironmentVariables.md [hunk 2]
```diff
@@ -21,3 +21,14 @@
...
+
+* `TARGET`: Set the database to use. If undefined, will assume SQLite.
+  Accepted values are:
+  * `SQLite`
+  * `PostgreSQL`
+  * `MySQL`
+
+* `ZONEMASTER_RECORD`: If set, the data from the test is recorded to a file.
+  Otherwise the data is loaded from a file.
```
Comment (matsduf) [inline docs/EnvironmentVariables.md:34]: Is "set" correct here? Shouldn't it be "if true"? If set to 0 or empty string will give the same affect as unset, won't it?

File: docs/EnvironmentVariables.md [hunk 3]
```diff
@@ -21,3 +21,14 @@
 * `ZM_BACKEND_RPCAPI_LOGJSON`: Setting it to any thruthy value (non-empty
   string or non-zero number) will configure the logger to log in JSON format,
   undefined by default.
+
+## Variables specific to testing
+
+* `TARGET`: Set the database to use. If undefined, will assume SQLite.
```
Comment (matsduf) [inline docs/EnvironmentVariables.md:27]: If think the following is more correct and better language:
```
If empty or undefind, "SQLite" will be assumed.
```

**Review by matsduf — CHANGES_REQUESTED** [review]

File: docs/EnvironmentVariables.md (hunk 1 above)
Comment (pnax) [reply docs/EnvironmentVariables.md]: updated

File: docs/EnvironmentVariables.md (hunk 2 above)
Comment (pnax) [reply docs/EnvironmentVariables.md]: fixed

File: docs/EnvironmentVariables.md (hunk 3 above)
Comment (pnax) [reply docs/EnvironmentVariables.md]: updated

**Review by matsduf — APPROVED** [review]
It is great to have to documented.

**Pull request merged**